sdk/mixedreality/Azure.MixedReality.Authentication/src/MixedRealityStsClientOptions.cs

```diff
@@ -20,11 +20,11 @@ public class MixedRealityStsClientOptions : ClientOptions
         /// Initializes a new instance of the <see cref="MixedRealityStsClientOptions"/> class.
         /// </summary>
         /// <param name="version">The version.</param>
-        public MixedRealityStsClientOptions(ServiceVersion version = ServiceVersion.V2019_02_28_Preview)
+        public MixedRealityStsClientOptions(ServiceVersion version = ServiceVersion.V2019_02_28)
         {
             Version = version switch
             {
-                ServiceVersion.V2019_02_28_Preview => "2019-02-28-preview",
+                ServiceVersion.V2019_02_28 => "2019-02-28-preview",
```

Comment:
V2019_02_28_preview should also map to "2019-02-28-preview", otherwise it's a breaking change.

Reply:
Both enum values are both mapped to the value of `1`, so the pattern matching treats them the same. In fact, you get an error if you try to handle them both.

Reply:
Ah, I missed that. Mapping two enum values to the same numeric value is a bit unusual, but I understand why you are doing this.